tests/molecules/FeCO6_b3lyp_pyscf/CMakeLists.txt

```diff
@@ -1,46 +1,50 @@
 
 IF (NOT QMC_CUDA)
  IF (NOT QMC_COMPLEX)
+  IF (ENABLE_SOA)
 #
-# LiH molecular dimer gaussian tests, all electron
+# [Fe(CO)6]2+ molecule gaussian tests, with BFD ECP
 # Also check results for different number of mpi tasks and threads keeping total constant
-#
-  LIST(APPEND FeCO6_PYSCF_SCALARS "kinetic" "165.48 0.36") # kinetic energy
-  LIST(APPEND FeCO6_PYSCF_SCALARS "totenergy" "-250.8138 0.0015") # total energy
-  LIST(APPEND FeCO6_PYSCF_SCALARS "eeenergy" "595.54 0.11") # e-e energy
-  LIST(APPEND FeCO6_PYSCF_SCALARS "samples" "32000 0.0") # samples
-#  LIST(APPEND LIH_SCALARS "flux" "0.0 0.1") # sampling check, should be zero
 
-  QMC_RUN_AND_CHECK(long-FeCO6_pyscf-vmc_b3lyp_noj
-                    "${CMAKE_SOURCE_DIR}/tests/molecules/FeCO6_b3lyp_pyscf"
-                    vmc_b3lyp_noj
-                    vmc_b3lyp_noj.in.xml
-                    16 1
-                    TRUE
-                    0 FeCO6_PYSCF_SCALARS # VMC
-                    )
+# Ref for long VMC test
+  LIST(APPEND FeCO6_PYSCF_SCALARS "kinetic" "163.8420 0.0022") # kinetic energy
```

Comment:
Are these scalars used?

Reply:
I preferred to leave the scalars from the reference run.

Reply:
Looks like the PYSCF_SCALARS can go...

Reply:
Keep the reference number is good.
Please change lines from `LIST(APPEND FeCO6_PYSCF_SCALARS "kinetic" "165.48 0.36") # kinetic energy`
To simply `# kinetic energy 165.48 0.36`